packages/webostv/template/package.json

```diff
@@ -35,7 +35,7 @@
   "dependencies": {
     "@enact/core": "^4.0.0-alpha.1",
     "@enact/i18n": "^4.0.0-alpha.1",
-    "@enact/moonstone": "^4.0.0-alpha.1",
+    "@enact/sandstone": "^2.0.0-alpha.1",
     "@enact/spotlight": "^4.0.0-alpha.1",
     "@enact/ui": "^4.0.0-alpha.1",
     "@enact/webos": "^4.0.0-alpha.1",
```

Comment:
Let's change enact module version to "^4.0.0" in advance.

Reply:
OK
Are these need to delete "-alpha.1", too?: dev-utils, cli